blockchain/state/statedb.go

```diff
@@ -473,6 +473,15 @@ func (s *StateDB) SetCode(addr common.Address, code []byte) error {
 	return nil
 }
 
+func (s *StateDB) SetCodeToEOA(addr common.Address, code []byte) error {
```

Comment:
Doesn't 7702 want SetCodehash(addr, codehash) because you want to set codeHash = delegation designator. cc @Mdaiki0730

Reply:
Oh I see. Thanks